doc/api/stream.md

```diff
@@ -1084,6 +1084,8 @@ buffer will be returned.
 If the `size` argument is not specified, all of the data contained in the
 internal buffer will be returned.
 
+The `size` argument must be less than or equal to 2^30.
```

Comment:
Nit: I would spell that for humans, i.e. 1 GiB.

Reply:
I think 1 GB is actually more correct in this case. I'll update.